packages/core/src/transport/WsOutboundTransporter.ts

```diff
@@ -3,22 +3,27 @@ import type { Logger } from '../logger'
 import type { ConnectionRecord } from '../modules/connections'
 import type { OutboundPackage } from '../types'
 import type { OutboundTransporter } from './OutboundTransporter'
+import type WebSocket from 'ws'
 
-import { InjectionSymbols } from '../constants'
-import { AriesFrameworkError } from '../error'
-import { WebSocket } from '../utils/ws'
+import { AgentConfig } from '../agent/AgentConfig'
+import { AriesFrameworkError } from '../error/AriesFrameworkError'
 
 export class WsOutboundTransporter implements OutboundTransporter {
   private transportTable: Map<string, WebSocket> = new Map<string, WebSocket>()
   private agent!: Agent
   private logger!: Logger
+  private WebSocket!: typeof WebSocket
 
   public supportedSchemes = ['ws', 'wss']
 
   public async start(agent: Agent): Promise<void> {
     this.agent = agent
-    this.logger = agent.injectionContainer.resolve(InjectionSymbols.Logger)
+
+    const agentConfig = agent.injectionContainer.resolve(AgentConfig)
+
+    this.logger = agentConfig.logger
     this.logger.debug('Starting WS outbound transport')
+    this.WebSocket = agentConfig.agentDependencies.WebSocket
```

Comment:
The `this.WebSocket` could be hard to understand what it means 🤔  like what's actually behind that property. Just saying, don't have any idea for improvement right now. 😄

Reply:
I've renamed to `WebSocketClass` for now. Let me know if you want something else